doc/man7/EVP_MAC-KMAC.pod

```diff
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-EVP_MAC-KMAC, EVP_MAC-KMAC256, EVP_MAC-KMAC256
+EVP_MAC-KMAC, EVP_MAC-KMAC256
```

Comment:
Please add `EVP_MAC-KMAC128`

Reply:
ah, that was probably the original intent. :)  fixed.